src/sentry/tasks/update_user_reports.py

```diff
@@ -35,7 +35,10 @@ def update_user_reports(**kwargs: Any) -> None:
     updated_reports = 0
     samples = None
 
-    MAX_EVENTS = kwargs.get("max_events", 5000)
+    MAX_EVENTS = kwargs.get(
+        "max_events",
+        4000,  # the default max_query_size is 256 KiB, this adds 40b per event, so 4000 is 160 KiB
```

Comment:
Each event id is 36 characters. I'm not sure that each character is just one byte though? If it was, I don't think we'd have a problem here, since we'd only have 200Kib, and 56Kib should be more than enough for the rest of the query.

I think we'd be fine here just going for chunks of 1-2k. It just means a few more queries, but it's not high volume

Reply:
It's `f"'{event.id}', "` so 36 bytes for ID plus the quotes, comma, and space = 40B. For 5000 that should be 200KiB, so not sure where the additional 56KiB comes from. AFAIK, ClickHouse uses UTF-8 encoding, so ASCII chars = 1B. 

Unless there's some additional quoting/encoding, such as base64 or URL encoding.

Reply:
anyways, reduced to `2000` just in case